Assets/HoloToolkit/UX/Scripts/Lines/Bezeir.cs

```diff
@@ -0,0 +1,132 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using UnityEngine;
+
+namespace MRTK.UX
+{
+    public class Bezeir : LineBase
+    {
+        [Serializable]
+        public struct PointSet
```

Comment:
If we're referencing this `PointSet` outside of `Bezeir` we should move it into it's own class, otherwise it should be private.

Reply:
I've made them private. The only other class that needs to know about them is the Bezier editor.